### PR TITLE
Add scatter to hybrid and make explicit

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

File: test/00-import.py [hunk 1]
```diff
@@ -1,4 +1,28 @@
 #
+# Copyright 2019-2019 Ghent University
```
Comment (stdweird) [inline test/00-import.py]: add a (empty) docstring here, so the header is not duplicated

File: test/__init__.py [hunk 2]
```diff
@@ -1,4 +1,28 @@
 #
+# Copyright 2019-2019 Ghent University
```
Comment (stdweird) [inline test/__init__.py]: same docstring remark

File: lib/vsc/mympirun/mpi/intelmpi.py [hunk 3]
```diff
@@ -173,7 +176,7 @@ def pinning_override(self):
         if self.pinning_override_type in ('packed', 'compact', 'bunch'):
             cmd.add(['-env', 'I_MPI_PIN_PROCESSOR_LIST=allcores:map=bunch'])
         elif self.pinning_override_type in ('spread', 'scatter'):
-            cmd.add(['-env', 'I_MPI_PIN_PROCESSOR_LIST=allcores'])
+            cmd.add(['-env', 'I_MPI_PIN_PROCESSOR_LIST=allcores:map=%s' % self.pinning_override_type])
```
Comment (stdweird) [inline lib/vsc/mympirun/mpi/intelmpi.py]: what are the possible values?

**Comment by stdweird** [comment]
@wpoely86 version bump?

File: lib/vsc/mympirun/mpi/intelmpi.py (hunk 3 above)
Comment (wpoely86) [reply lib/vsc/mympirun/mpi/intelmpi.py]: https://software.intel.com/en-us/mpi-developer-reference-windows-environment-variables-for-process-pinning

bunch, spread and scatter

**Comment by wpoely86** [comment]
The version bump is well hidden between all the style stuff :wink:

File: lib/vsc/mympirun/mpi/intelmpi.py (hunk 3 above)
Comment (boegel) [reply lib/vsc/mympirun/mpi/intelmpi.py]: tests?

**Pull request merged**